src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/ace-uncompressed.js

```diff
@@ -9704,8 +9704,6 @@ var TextHighlightRules = require("./text_highlight_rules").TextHighlightRules;
 
```

Comment:
ace-uncompressed is still showing changes, we should back these all the way out.